Shape_detection/doc/Shape_detection/Shape_detection.txt

```diff
@@ -364,41 +366,45 @@ to the quality of the least squares plane fit applied to the neighbors of each p
 
 The classes in Section \ref Shape_detection_RegionGrowingPoints "Region Growing On Point Set" depend on a few parameters
 that should be defined by the user. They also have default values, but these values do not necessarily guarantee
-to produce pleasant results.
+to produce pleasant results. All parameter values can be specified using \ref bgl_namedparameters "Named Parameters".
 
-The `NeighborQuery` related classes depend on the following parameters:
+The `NeighborQuery` related classes depend on the `neighbor_radius` parameter:
 
-- `sphere_radius` is used by the class `CGAL::Shape_detection::Point_set::Sphere_neighbor_query`
-and defines the radius of the fuzzy search sphere centered at the query point;
+- `neighbor_radius` in the class `CGAL::Shape_detection::Point_set::Sphere_neighbor_query`,
+defines the radius of the fuzzy search sphere centered at the query point;
 
-- `k` is used by the class `CGAL::Shape_detection::Point_set::K_neighbor_query`
-and defines the number K of nearest neighbors of the query point.
+- `neighbor_radius` in the class `CGAL::Shape_detection::Point_set::K_neighbor_query`,
+defines the number K of nearest neighbors of the query point.
```

Comment:
This change seems weird: the previous version was clearer, if I have a class named `K_neighbor_query`, the parameter `k` is expected and quite understandable. Now it's `neighbor_radius`, but in practise it's not a radius. This seems to go against the principle of least surprise. I understand it's to have a unified naming, but it seems like a bad reason to change that.

Reply:
Honestly, I also do not like very much this name but I tried to have the same unified name in the named parameters of different neighbor classes. I will see how to change it to satisfy the principle of least surprise. You are absolutely right!

Reply:
I now use `min` and `max` uniformly.